src/User32/User32+SendMessageTimeoutFlags.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    /// <content>
+    /// Contains the <see cref="SendMessageTimeoutFlags"/> nested type.
+    /// </content>
+    public partial class User32
+    {
+        /// <summary>
+        /// Possible flag values for <see cref="User32.SendMessageTimeout"/>
+        /// Documentation pulled from MSDN.
```

Comment:
Nitpick: That can be removed, we pull all documentation from MSDN :)

Reply:
I removed it.